extensions/spring-cloud-config-client/runtime/src/main/java/io/quarkus/spring/cloud/config/client/runtime/SpringCloudConfigClientRecorder.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.spring.cloud.config.client.runtime;
+
+import java.util.Collections;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+import org.eclipse.microprofile.config.spi.ConfigSourceProvider;
+import org.jboss.logging.Logger;
+
+import io.quarkus.runtime.ApplicationConfig;
+import io.quarkus.runtime.RuntimeValue;
+import io.quarkus.runtime.annotations.Recorder;
+import io.quarkus.runtime.configuration.ProfileManager;
+
+@Recorder
+public class SpringCloudConfigClientRecorder {
+
+    private static final Logger log = Logger.getLogger(SpringCloudConfigClientRecorder.class);
+
+    public RuntimeValue<ConfigSourceProvider> create(SpringCloudConfigServerClientConfig springCloudConfigServerClientConfig,
+            ApplicationConfig applicationConfig) {
+        if (!springCloudConfigServerClientConfig.enabled) {
+            log.debug(
+                    "No attempt will be made to obtain configuration from the Spring Cloud Config Server because the functionality has been disabled via configuration");
+            return new RuntimeValue<>(new EmptyConfigSourceProvider());
+        }
+
+        if (!springCloudConfigServerClientConfig.baseUri.isPresent()) {
+            log.warn(
+                    "No attempt will be made to obtain configuration from the Spring Cloud Config Server because the baseUri has not been set. Considering setting it via 'quarkus."
```

Comment:
```suggestion
                    "No attempt will be made to obtain configuration from the Spring Cloud Config Server because the baseUri has not been set. Consider setting it via 'quarkus."
```

Reply:
Btw, nice try to put the typos outside of my review screen ;).

Reply:
One can only try :laughing: